packages/amplify-provider-awscloudformation/src/aws-utils/aws-s3.ts

```diff
@@ -136,7 +136,7 @@ export class S3 {
 
   async getAllObjectVersions(
     bucketName: string,
-    continuationToken: Required<Pick<ListObjectVersionsOutput, 'KeyMarker' | 'VersionIdMarker'>> = null,
+    continuationToken: Pick<ListObjectVersionsOutput, 'KeyMarker' | 'VersionIdMarker' | 'Prefix'> = null,
```

Comment:
Why adding `Prefix` breaking the `Required`?

Reply:
Since this function is used by two other functions
- One to delete the whole bucket
- One to delete a "directory"

Both don't need `Prefix`, changing the type to mainly require the first `KeyMarker` and `VersionIdMarker` if used.